webrtc/videoroom/assets/src/membraneWebRTC.ts

```diff
@@ -104,14 +109,28 @@ export class MembraneWebRTC {
 
     this.channel.on("offer", this.onOffer);
     this.channel.on("candidate", this.onRemoteCandidate);
+    this.channel.on("replaceTrack", (data: any) => {
+      const oldTrackId = data.data.oldTrackId;
```

Comment:
are you sure typescript is not complaining about the any type here?

Reply:
I don't see any warnings